google/cloud/storage/benchmarks/aggregate_download_throughput_benchmark.cc

```diff
@@ -120,21 +119,17 @@ class Iteration {
 };
 
 gcs::Client MakeClient(AggregateDownloadThroughputOptions const& options) {
-  auto client_options =
-      google::cloud::Options{}
-          .set<gcs_experimental::HttpVersionOption>(options.rest_http_version)
-          .set<gcs::TransferStallTimeoutOption>(options.download_stall_timeout);
+  auto opts = google::cloud::Options{options.client_options}
+                  // Make the upload buffer size small, the library will flush
```

Comment:
s/small, the/small so the/

Reply:
is setting `UploadBufferSizeOption` in the download benchmark an intentional change?

Reply:
Nope. Fixed.